### PR TITLE
Hotfix/external contributions wrong users

### PR DESCRIPTION
**Pull request by GuilleLeopold**

### CODE REVIEW

File: app/services/github_client/user.rb
Comment (SandroDamilano) [inline app/services/github_client/user.rb:14]: Would it make sense to return an empty list instead of `nil` here? (Or whatever Exception Hunter returns)
I think that way we wouldn't need to add [this line](https://github.com/rootstrap/rs-code-review-metrics/pull/344/files#diff-f5e889280495856ba8b2fd80796cc080R9).

**Review by SandroDamilano — APPROVED** [review]
Looks good!
I think it would be great to have this code on `develop` as well. What would be best, to open another PR pointing to develop or to point this one to develop and making a release immediately after it's merged? I personally prefer the second one, but I'll leave it up to you.
Also this is a reminder to open a task to add a test to this case 😁

File: app/services/github_client/user.rb
Comment (horacio) [inline app/services/github_client/user.rb:24]: WDYT about using `flat_map` here, so we can just do `events.select { |event| event[:type] == 'PullRequestEvent' }` in `#pull_request_events`?

File: app/services/processors/external/pull_requests.rb
Comment (horacio) [inline app/services/processors/external/pull_requests.rb:9]:
```suggestion
        return unless events.present?
```

**Review by horacio — APPROVED** [review]
👏

File: app/services/github_client/user.rb
Comment (GuilleLeopold) [reply app/services/github_client/user.rb:14]: Yes. It seems like ExceptionHunter returns nil. I thought about doing that too. It might be fine

**Comment by GuilleLeopold** [comment]
I was talking with Martin and i think the best is to merge master into develop after merging this one

**Pull request merged**
